.eslintrc.js

```diff
@@ -63,59 +63,57 @@ module.exports = {
                 ignoredNodes: ['JSXElement *']
             }
         ],
-        'key-spacing': [2, { beforeColon: false, afterColon: true }],
-        'keyword-spacing': [2, { before: true, after: true }],
-        'linebreak-style': 0,
-        'max-len': [2, 200, 2, { ignoreUrls: true, ignoreRegExpLiterals: true }],
-        'newline-after-var': 2,
-        'require-atomic-updates': 0,
-        'no-constant-condition': 2,
-        'no-dupe-class-members': 2,
-        'no-lonely-if': 2,
-        'no-multiple-empty-lines': [2, {
-            max: 1,
-            maxEOF: 0
-        }],
-        'no-prototype-builtins': 0,
-        'no-underscore-dangle': 2,
-        'no-var': 2,
-        'object-curly-spacing': [2, 'always'],
-        'object-curly-newline': [2, { consistent: true }],
-        'object-property-newline': [2, { allowAllPropertiesOnSameLine: true }],
-        'object-shorthand': [2, 'methods'],
-        'operator-linebreak': [0, 'before'],
-        'padded-blocks': [2, 'never'],
+        'key-spacing': ['error', { beforeColon: false, afterColon: true }],
+        'keyword-spacing': ['error', { before: true, after: true }],
+        'linebreak-style': ['error', 'unix'], // no carriage returns
+        'max-len': ['error', { code: 120 }], // be friendly to laptops
```

Comment:
General theory, breaking lines can be harder to read.

- People naturally break lines when they're hard to read so max length has never been an issue and is usually
- reason why I go longer than 140 is for long strings so it doesn't eat up lines.
- Nobody has 1024x768 screens anymore XD. most are 1080p or 
- 140 for less than the default screen 
- 174 for default screen 
- 220 for philip's monitor XD
-         Recommendation: 140 seems ideal realistic length for a line to be. Longer is